lib/federation-utils.js

```diff
@@ -23,7 +23,81 @@ var compareFederateKeys = (publicKeysA, publicKeysB) => {
   return comparison;
 };
 
+const getActiveFederationPublicKeys = async (bridge) => {
```

Comment:
What do you think of using [powpeg-details](https://github.com/rsksmart/powpeg-details) library?

Not now, but if you think it's a good idea we can add it in a separate PR

Reply:
It only works for the active federation.